docs/developer/decision-records/2022-06-20-mvd-onboarding/README.md

```diff
@@ -0,0 +1,36 @@
+# Participants onboarding in MVD
+
+This document outlines the infrastructure steps for the scenario of onboarding participants to the Minimum Viable Dataspace (MVD).
+
+## Precondition - GAIA-X membership
+
+To join the MVD, each participant must have a Verifiable Credential (VC) signed by GAIA-X Authority which proves GAIA-X membership.
+
+At present the MVD deployment pipeline does not interact with the real GAIA-X Authority. A simulated GAIA-X Authority is deployed within MVD for demonstration purposes.
+
+## Onboarding process - GAIA-X membership verification
+
+During the onboarding process, the GAIA-X membership claim is verified by the Dataspace Authority.
+The onboarding flow is presented in the [distributed authorisation sub-flow document](https://github.com/agera-edc/MinimumViableDataspace/tree/feature/20-rs-adr-target/docs/developer/decision-records/2022-06-16-distributed-authorization).
+Actors in the onboarding scenario:
+
+- _Participant A_ is a putative participant that wants to join the MVD.
+- _Participant B_ is the Dataspace Authority (Registration Service and its `did:web` document).
```

Comment:
nit: We also have below actor named as "Authority" and Participant B is also called Datasapce Authority. So, to avoid naming conflict can we say "_Participant B_ is the Registration Service with its `did:web` document" ?

What do you think? Thanks

Reply:
agree, changed